apps/webapp/state/ibc.ts

```diff
@@ -0,0 +1,52 @@
+import { AllSlices, SliceCreator } from '.';
+import { Asset, Chain } from '../types/asset';
+import { validateAmount } from '../utils';
+
+export interface IbcValidationFields {
+  amount: boolean;
+}
+
+export interface IbcSlice {
```

Comment:
Let's call this `IbcSendSlice`. In the future, we may have more IBC-related slices.